devtools-ci-index.yaml

```diff
@@ -2994,6 +3083,34 @@
             ci_project: 'devtools'
             ci_cmd: '/bin/bash ./.ci/cico_build_prcheck_dep.sh'
             timeout: '120m'
+        - '{ci_project}-{git_repo}-periodic-route-{cluster}':
+            git_organization: redhat-developer
+            git_repo: rh-che
+            ci_project: 'devtools'
+            ci_cmd: './functional-tests/devscripts/route_tests.sh -u $USERNAME -p $PASSWORD -f ./functional-tests/devscripts/resources/route.yml'
```

Comment:
Please, do you think that $USERNAME and $PASSWORD which are saved in Vault can be accessed this way?

Reply:
if vault-keys are set correctly in the vault secret it should be ok.

Reply:
Ok, thank you :+1: